script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -0,0 +1,89 @@
+; Script generated by the Inno Script Studio Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Git LFS"
+#define MyAppVersion "1.1.0"
+#define MyAppPublisher "GitHub, Inc"
+#define MyAppURL "https://git-lfs.github.com/"
+#define MyAppFilePrefix "git-lfs-windows-amd64"
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application.
+; Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{286391DE-F778-44EA-9375-1B21AAA04FF0}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
+AppPublisher={#MyAppPublisher}
+AppPublisherURL={#MyAppURL}
+AppSupportURL={#MyAppURL}
+AppUpdatesURL={#MyAppURL}
+LicenseFile=..\..\LICENSE.md
+OutputBaseFilename={#MyAppFilePrefix}-{#MyAppVersion}
+OutputDir=..\..\
+Compression=lzma
+SolidCompression=yes
+DefaultDirName={code:GetExistingGitInstallation}
+UsePreviousAppDir=no
+DirExistsWarning=no
+DisableReadyPage=True 
+
+[Languages]
+Name: "english"; MessagesFile: "compiler:Default.isl"
+
+[Files]
+Source: ..\..\git-lfs.exe; DestDir: "{app}"; Flags: ignoreversion; AfterInstall: InstallGitLFS;
+
+[Code]
+// Uses cmd to parse and find the location of Git through the env vars
+function GetExistingGitInstallation(Value: string): string;
+var
+  TmpFileName: String;
+  ExecStdOut: AnsiString;
+  ResultCode: integer;
+
+begin      
+  TmpFileName := ExpandConstant('{tmp}') + '\git_location.txt';
+  
+  Exec(
+    ExpandConstant('{cmd}'),
+    '/C "for %i in (git.exe) do @echo. %~$PATH:i > "' + TmpFileName + '"', 
+    '', SW_HIDE, ewWaitUntilTerminated, ResultCode
+  );
```

Comment:
This is problematic because git installers typically setup an alternate shell shortcut, like "Git CMD". But the default `%PATH%` will not have Git anywhere. Not sure what we can do about that though.